fe/fe-core/src/main/java/com/starrocks/connector/ConnectorMetadata.java

```diff
@@ -264,5 +266,8 @@ default void createView(CreateViewStmt stmt) throws DdlException {
     default void alterView(AlterViewStmt stmt) throws DdlException, UserException {
     }
 
+    default CloudConfiguration getCloudConfiguration() {
```

Comment:
Using non-default implementation function better? This means we must implement it.

Reply:
I think it's more common to throw exception when using default

Reply:
Yes. updated.